test/components/views/rooms/RoomPreviewBar-test.tsx

```diff
@@ -71,8 +71,9 @@ describe("<RoomPreviewBar />", () => {
     const inviterUserId = "@inviter:test.com";
     const otherUserId = "@othertester:test.com";
 
-    const getComponent = (props: ComponentProps<typeof RoomPreviewBar> = {}) => {
+    const getComponent = (props: Omit<ComponentProps<typeof RoomPreviewBar>, "roomId"> = {}) => {
```

Comment:
given that `roomId` is now optional, I think this change is unnecessary.

Reply:
removed this usage and another one too